requirements.txt

```diff
@@ -26,5 +26,5 @@ cmsis-pack-manager>=0.2.3,<0.3.0
 pywin32==224;platform_system=='Windows'
 wmi==1.4.9;platform_system=='Windows'
 psutil==5.6.2
-cryptography>=2.4.x,<2.5
+cryptography>=2.5,<=2.8
```

Comment:
Is there a reason why the upper bound is limited to 2.8? Can we not use '<3'

Reply:
Fixed.